.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: false
 language: go
 
 go:
- - 1.6.3
```

Comment:
We just had a discussion about supported versions in #291 and the decision was to test against 1.6 but to just build tags for 1.7+ features.